topics/list_topics.go

```diff
@@ -0,0 +1,35 @@
+package topics
+
+import (
+	"context"
+	"regexp"
+
+	kafkago "github.com/segmentio/kafka-go"
+)
+
+// ListTopics returns a slice of all the Topics
+func ListTopics(ctx context.Context, client *kafkago.Client) (topics []kafkago.Topic, err error) {
+	response, err := client.Metadata(ctx, &kafkago.MetadataRequest{
+		Addr: client.Addr,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return response.Topics, nil
```

Comment:
What do you think about returning a slice of strings instead of `Topic`? If we do so we'll also want to check the `Topic.Error` before returning

Reply:
I initially had implemented this to return a slice of strings. But it seemed returning Topic fit better with your API. Why would you prefer strings?

Reply:
I was thinking that it might be less to maintain but yeah I agree that it's more general purpose this way, let's keep it `Topic` the way you have it